### PR TITLE
Use AutoWidth rather than requiring a width passed to MetricChartBlock.

### PR DESCRIPTION
**Pull request by mikelehen**

### CODE REVIEW

File: packages/ui-components/src/components/MetricChartBlock/MetricChartBlock.tsx [hunk 1]
```diff
@@ -103,12 +99,14 @@ export const MetricChartBlock = ({
...
-                  metric={metric}
-                  region={region}
-                  width={width}
-                  height={height}
-                />
+                <AutoWidth>
+                  <MetricChart
+                    metric={metric}
+                    region={region}
+                    width={/*let auto-width set it*/ 0}
```
Comment (mikelehen) [inline packages/ui-components/src/components/MetricChartBlock/MetricChartBlock.tsx:106]: This feels a little hacky, but I'm guessing it's the only way to make it work.  The other option is to use `ParentSize` directly.

**Review by chasulin — COMMENTED** [review]
this seems to work! (which is slightly confusing to me..[this line of documentation in AutoWidth](https://github.com/covid-projections/act-now-packages/blob/develop/packages/ui-components/src/components/AutoWidth/AutoWidth.tsx#L21-L22) needs to be adjusted, as i thought that the child having a hardcoded with would mean that AutoWidth's width wouldnt be applied). 

an idea- maybe we move the implementation of AutoWidth right into the chart files, so theyre always responsive. we did this for CAN charts ([example](https://github.com/covid-projections/covid-projections/blob/develop/src/components/Charts/ChartCaseDensity.tsx#L180-L199)) and it seemed to work out well-- it makes the charts responsive by default, and if we ever want a fixed-width/non-responsive chart, we could just wrap the implementation in a Box with a declared width.

cc @pnavarrc - thoughts?

**Comment by mikelehen** [comment]
@chasulin Oops, I should have updated that comment.  My code changes it so `AutoWidth` will still apply its width if width is set to zero in the child component.

**Comment by chasulin** [comment]
> @chasulin Oops, I should have updated that comment. My code changes it so `AutoWidth` will still apply its width if width is set to zero in the child component.

ooooh i see, got it. thanks for clarifying 🙏🏼

**Comment by pnavarrc** [comment]
@chasulin I think we need to explore a few alternatives, because `AutoWidth` doesn't work well with containers that don't have an intrinsic width (flex boxes), and having children imposing restrictions on parent (the parent needs to have a width) components is kind of an anti-pattern.

I'm going to outline a few options so we can discuss during standup.

**Review by pnavarrc — APPROVED** [review]
Looks good! (thanks @chasulin for reviewing too)

**Comment by chasulin** [comment]
@pnavarrc yeah, i meant wrapping charts in AutoWidth (which are composed svg's -- not all components, and not components that use flexbox/dont have intrinsic width). so, similar to that CAN example that I linked above ([here](https://github.com/covid-projections/covid-projections/blob/develop/src/components/Charts/ChartCaseDensity.tsx#L180-L199))

**Pull request merged**